### ISSUE TITLE
input keyword not displaying text or accepting response

### ISSUE THREAD

**Issue by seanjmcm** [spyder-ide/spyder#17806]
### Issue Report Checklist

* [x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [x] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [x] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [x] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [x] Tried basic troubleshooting (if a bug/error)
    * [x] Restarted Spyder
    * [x] Reset preferences with ``spyder --reset``
    * [ ] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [x] Tried the other applicable steps from the Troubleshooting Guide
* [x] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description

input text does not appear and does not exit after enter is pressed.

### What steps reproduce the problem?

1. name = input("what is your name? \n")
2. 
3. 

### What is the expected output? What do you see instead?
expected : what is your name?
actual: nothing hangs at runfile('D:/x/age.py', wdir='D:/x/week 2')

### Paste Traceback/Error Below (if applicable)

```python-traceback

Traceback (most recent call last):

  File D:\x\age.py:9 in <module>
    name = input("what is your name? \n")

  File d:\ProgramData\Anaconda3\lib\site-packages\ipykernel\kernelbase.py:1075 in raw_input
    return self._input_request(

  File d:\ProgramData\Anaconda3\lib\site-packages\ipykernel\kernelbase.py:1120 in _input_request
    raise KeyboardInterrupt("Interrupted by user") from None

KeyboardInterrupt: Interrupted by user

```

## Versions

* Spyder version: 5.1.5
* Python version: 3.8.8
* Qt version: 5.9.7
* PyQt version: 5.9.2
* Operating System name/version: Windows 10 

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  2.0.0 (OK)
cookiecutter >=1.6.0          :  1.7.3 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0               :  8.2.0 (OK)
jedi >=0.17.2;<0.19.0         :  0.17.2 (OK)
jsonschema >=3.2.0            :  3.2.0 (OK)
keyring >=17.0.0              :  23.4.0 (OK)
nbconvert >=4.0               :  6.4.4 (OK)
numpydoc >=0.6.0              :  1.2 (OK)
paramiko >=2.4.0              :  2.8.1 (OK)
parso >=0.7.0;<0.9.0          :  0.7.0 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.8.0 (OK)
pygments >=2.0                :  2.11.2 (OK)
pylint >=2.5.0;<2.10.0        :  2.7.4 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.2.2;<1.3.0          :  1.2.4 (OK)
pylsp_black >=1.0.0           :  None (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.1.10 (OK)
qtawesome >=1.0.2             :  1.0.3 (OK)
qtconsole >=5.1.0             :  5.3.0 (OK)
qtpy >=1.5.0                  :  2.0.1 (OK)
rtree >=0.9.7                 :  0.9.7 (OK)
setuptools >=49.6.0           :  61.2.0 (OK)
sphinx >=0.6.6                :  4.4.0 (OK)
spyder_kernels >=2.1.1;<2.2.0 :  2.1.3 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3             :  2.1.6 (OK)
zmq >=17                      :  22.3.0 (OK)

# Optional:
cython >=0.21                 :  0.29.28 (OK)
matplotlib >=2.0.0            :  3.5.1 (OK)
numpy >=1.7                   :  1.20.3 (OK)
pandas >=1.1.1                :  1.4.1 (OK)
scipy >=0.17.0                :  1.7.3 (OK)
sympy >=0.7.3                 :  1.10.1 (OK)
```

**Comment by ccordoba12** [comment]
Hey @seanjmcm, thanks for reporting. This error should be fixed in our latest version (5.3.0).

Since it's still not available in Anaconda, you have two options:

1. Use our Windows installer, which you can find [here](https://github.com/spyder-ide/spyder/releases).
2. Create a new environment with conda-forge packages. For that, please close Spyder, open the Anaconda Prompt and run the following commands there:

        conda create -n spyder-cf -c conda-forge spyder
        conda activate spyder-cf
        spyder

Let us know if that works for you.

**Comment by thetoegi** [comment]
> Hey @seanjmcm, thanks for reporting. This error should be fixed in our latest version (5.3.0).
> 
> Since it's still not available in Anaconda, you have two options:
> 
>     1. Use our Windows installer, which you can find [here](https://github.com/spyder-ide/spyder/releases).
> 
>     2. Create a new environment with conda-forge packages. For that, please close Spyder, open the Anaconda Prompt and run the following commands there:
>        ```
>         conda create -n spyder-cf -c conda-forge spyder
>         conda activate spyder-cf
>         spyder
>        ```
> 
> 
> Let us know if that works for you.

I do not know if it worked for him, but I used the second option and it worked for me. 

Thank you very much.

**Comment by seanjmcm** [comment]
Sorry, I didn't realize this had gone through.

I followed the reinstall anaconda tick as follows:

`conda install anaconda=2021.11
`
This cleared the issue and input worked again.
